src/sortStudents.ts

```diff
@@ -1,16 +1,56 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averageGrade',
 }
 
-// create SortOrder type
-export type SortOrder;
+export type SortOrder = 'asc' | 'desc';
 
+export function sortStudents(
+  students: Student[],
+  sortBy: SortType,
+  order: SortOrder,
+): Student[] {
+  function setOrder(): number {
+    return order === 'asc' ? 1 : -1;
+  }
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+  type Compare = string | number | boolean;
+
+  function compare(x: Student): Compare {
```

Comment:
Don't declare types inside the function. And you don't really need a separate type here. Improve variable names 
```suggestion
  function compare(student: Student): string | number {
```